drivers/SmartThings/sonos/src/sonos_state.lua

```diff
@@ -368,7 +388,14 @@ function SonosState:get_coordinator_for_group(household_id, group_id)
         false
       )
   end
-  return household.groups[group_id].coordinatorId
+
+  local group = (household.groups or {})[group_id]
+  if type(group) ~= "table" then
+    log.error(string.format("No known group for id %s in household %s", group_id, household_id))
+    return
```

Comment:
Do you want these error logs to go into hub logs? We dont expect this to happen, but if it is happening and we are investigating a hub, it could be a red flag to look into.